common/CommunityToolkit.Labs.Shared/Renderers/Markdown/MarkdownTextBlock.cs

```diff
@@ -16,11 +16,79 @@
     #endif
 #endif
 
+#if __WASM__
+using Markdig;
```

Comment:
@Arlodotexe this seems weird?

![image](https://user-images.githubusercontent.com/24302614/201175004-0284b67b-16e1-4926-b42a-da3f4003257a.png)

Why is it trying to find `Markdig` here in the `__WASM__` guard for Mac, iOS, and Android?

Reply:
We manually define these in `./common/Labs.ProjectIdentifiers.props`. It may be that we have a bad condition in that file, causing `IsWasm` to be set to `true` under Droid, macOS and iOS.

Reply:
@Arlodotexe looks like this just may be a VS issue. CI seemed fine. Not sure what that means for building those heads locally though.